### DISCUSSION TITLE
Install guide

### DISCUSSION THREAD

**Discussion by xansone** [screego/server#135] [General]
Good afternoon, 
Probably I am annoying person and I truly apologize but I am just newbie, googling how to do some everything on servers. I am providing small server for team speak with friends, we are missing only screen sharing possibility, your screego would fit perfect for it. I tried to install it but as I said I am newbie and cant truuly make it work, it starts but only termporaily and also have TSL problems, is there maybe any guide with details how to run it smoothly?

**Comment by jmattheis** [comment]
You can find the installation documentation here https://screego.net/#/install

There are lots of tutorials how to set up TLS on the OS that you are using. If this doesn't work for you, then you can use the public instance hosted at https://app.screego.net/
